docs/cristin.ttl

```diff
@@ -514,14 +542,98 @@
     :mapstonvafield "entityDescription.contributor.role. only the first role of a person is considered." .
 
 :rollekode
-    a            rdf:Property ;
-    rdfs:comment """
+    a               :cristinfield ;
+    rdfs:comment    """
         The role of the respective Contributor for the respective publication affiliated for this role with the respective institution.
         This entry is immutable and does not change in time.
         """ ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :VARBEID_PERSON_STED_ROLLE_Class ;
+    :isignored      "false" ;
+    :mapstonvafield " maps to NVA Role" .
+
+:eierkode
+    a            :cristinfield ;
     rdfs:range   xsd:string ;
-    rdfs:domain  :VARBEID_PERSON_STED_ROLLE_Class ;
-    :isignored   "false" .
+    rdfs:domain  :VITENSKAPELIGARBEID_LOKAL_Class ;
+    rdfs:comment """The institution the currently owns the NVI result. Example UIO.
+    We do not know whether the values of this field are standardized and whether there is a defined set of values for this property's range""" ;
+    :isignored   "true" .
+
+
+:status_kontrollert
+    a            :cristinfield ;
+    rdfs:range   xsd:string ;
+    rdfs:domain  :VITENSKAPELIGARBEID_LOKAL_Class ;
+    rdfs:comment """
+     It indicates if the respective post has been controlled or not. It receives values "J", "N".
+     It is unknown whether it refers to the Cristin entry or the NVI entry.
```

Comment:
It is part of the NVI entry, so it refers to the respective institution's opinion on whether the Cristin entry should be reported to NVI and count towards RBK funding.

NULL = this institution has not yet given their opinion
J = this institution believes the current result should be included in the NVI report
N = this institution believes the current result should NOT be included in the NVI report (e.g. doesn't meet academic standards, or has other issues)

In the period up to generating the yearly report to NVI, the relevant institutions should ping-pong until they agree on J or N.